docs/extending.md

```diff
@@ -318,6 +317,45 @@ class CustomDynamicGroups(BaseDynamicGroups):
 
 Now if the `context` looks like `{"coast": "west", "additional_groups": ["board", "administration"]}`, the resulting groups are `["legal-dept-san-francisco", "legal-dept-seattle", "board", "administration"]`.
 
+## Dynamic Tasks
+
+Caluma allows you to extend the JEXL evaluation in the `next` property of a
+flow. This `next` property decides which task(s) will be started on
+completing or skipping a work item.
+
+This concept can be used to implement an exclusive choice flow. E.g if the
+document of the case has a certain answer it will start a different task than
+normally.
+
+To configure a dynamic tasks class, you can set the `DYNAMIC_TASKS_CLASSES` environment variable.
+When using the Caluma service, you can place your class in `caluma.extensions.dynamic_tasks` and set `DYNAMIC_TASKS_CLASSES` to `["caluma.extensions.dynamic_tasks.CustomDynamicTasks"]`.
+
+### Example
+
+```python
+from caluma.caluma_workflow.dynamic_tasks import (
+    BaseDynamicTasks,
+    register_dynamic_task,
+)
+
+
+class CustomDynamicTasks(BaseDynamicTasks):
+    @register_dynamic_task("audit-forms")
+    def resolve_audit_forms(self, case, user, prev_work_item, context):
+        needs_extra_audit = case.document.answers.filter(question_id="is-extra-complicated", value="is-extra-complicated-yes").exists()
```

Comment:
```suggestion
        needs_extra_audit = case.document.answers.filter(
            question_id="is-extra-complicated",
            value="is-extra-complicated-yes"
        ).exists()
```

Reply:
Black doesn't format python inside markdown snippets :-1: I'm used to prettier doing my formatting inside markdown files as well :wink:

Reply:
Fixed